src/index.js

```diff
@@ -128,24 +131,28 @@ export default class Resizable extends Component {
       const min = (minWidth < 0 || typeof minWidth === 'undefined') ? 0 : minWidth;
       const max = (maxWidth < 0 || typeof maxWidth === 'undefined') ? newWidth : maxWidth;
       newWidth = clamp(newWidth, min, max);
+      newWidth = snap(newWidth, this.props.grid[0]);
     }
     if (/left/i.test(direction)) {
       newWidth = original.width - clientX + original.x;
       const min = (minWidth < 0 || typeof minWidth === 'undefined') ? 0 : minWidth;
       const max = (maxWidth < 0 || typeof maxWidth === 'undefined') ? newWidth : maxWidth;
       newWidth = clamp(newWidth, min, max);
+      newWidth = snap(newWidth, this.props.grid[0]);
     }
     if (/bottom/i.test(direction)) {
       newHeight = original.height + clientY - original.y;
       const min = (minHeight < 0 || typeof minHeight === 'undefined') ? 0 : minHeight;
       const max = (maxHeight < 0 || typeof maxHeight === 'undefined') ? newHeight : maxHeight;
       newHeight = clamp(newHeight, min, max);
+      newWidth = snap(newWidth, this.props.grid[1]);
     }
```

Comment:
That is a mistake.

correct?
`newHeight = snap(newHeight, this.props.grid[1]);`